public-pool/umbrel-app.yml

```diff
@@ -0,0 +1,26 @@
+manifestVersion: 1.1
+id: public-pool
+category: bitcoin
+name: Public Pool
+version: "4bb00f2"
+tagline: Fully Open Source Solo Bitcoin Mining Pool
+description: >-
+  Fully Open Source Solo Bitcoin Mining Pool
+
+
+  Don't trust. Verify. On steroids.
```

Comment:
Should we add anything here about connecting a nerdminer, bitaxe, etc? Or just leave as-is?

Reply:
nah they've all got their own set-up guides, so i didn't think it would be worth adding. but if anyone reaches out asking what URL to use i could add a little extra to the description.